frontend/src/concepts/roleBinding/RoleBindingPermissionsNameInput.tsx

```diff
@@ -48,33 +47,27 @@ const RoleBindingPermissionsNameInput: React.FC<RoleBindingPermissionsNameInputP
     );
   }
 
+  const selectOptions = typeAhead.map((option) => ({
+    value: isProjectSubject ? namespaceToProjectDisplayName(option, projects) : option,
+    content: isProjectSubject ? namespaceToProjectDisplayName(option, projects) : option,
+  }));
```

Comment:
is content necessary? won't it just render the value as content? if not, compute only once.
```suggestion
  const selectOptions = typeAhead.map((option) => {
    const value = isProjectSubject ? namespaceToProjectDisplayName(option, projects) : option;
    return {
      value,
      content: value,
    };
  });
```